scripts/tcl_commands/routing.tcl

```diff
@@ -423,31 +410,16 @@ proc run_routing {args} {
         ins_fill_cells
     }
 
-    set global_routed_netlist [index_file $::env(routing_tmpfiles)/global.v]
-    write_verilog $global_routed_netlist -log $::env(routing_logs)/write_verilog_global.log
-    set_netlist $global_routed_netlist
-    if { $::env(LEC_ENABLE) } {
-        logic_equiv_check -rhs $::env(PREV_NETLIST) -lhs $::env(CURRENT_NETLIST)
-    }
+    write_verilog\
+        [index_file $::env(routing_tmpfiles)/grt.nl.v]\
+        -powered_to [index_file $::env(routing_tmpfiles)/grt.pnl.v]\
+        -log $::env(routing_logs)/write_verilog_global.log
 
     # detailed routing
     detailed_routing
 
-    set detailed_routed_netlist [index_file $::env(routing_tmpfiles)/detailed.v]
-
-    write_verilog $detailed_routed_netlist -log $::env(routing_logs)/write_verilog_detailed.log
```

Comment:
was this accidentally removed or is it part of somewhere else or perhaps not needed ?